recipe/meta.yaml

```diff
@@ -32,6 +33,7 @@ test:
     - marshmallow_sqlalchemy
   requires:
     - pip
+    - python <3.10
```

Comment:
I'm not sure we want to restrict just testing to `<3.10`. Testing env should match production as closely as possible.

Reply:
Without python <3.10 it test/requires the tests can fail for noarch packages, because of conda issue